### PR TITLE
Add save/revert/close commands

### PR DESCRIPTION
**Pull request by blink1073**
Also updates to latest dependencies.

screen shot 2016-02-03 at 4 08 31 pm

### CODE REVIEW

File: src/documentmanager/plugin.ts [hunk 1]
```diff
@@ -131,6 +138,134 @@ function resolve(container: Container): Promise<void> {
...
+      registry.add([
+        {
+          id: saveDocumentId,
+          command: saveDocumentCommand
+        }
+      ]);
+      shortcuts.add([
+        {
+          sequence: ['Accel S'],
+          selector: '.' + DOCUMENT_CLASS,
```
Comment (jasongrout) [inline src/documentmanager/plugin.ts]: Is ``.${DOCUMENT_CLASS}`` easier to read?

Comment (jasongrout) [inline src/documentmanager/plugin.ts]: this shortcut can also have a selector for the active document, right?

File: src/imagehandler/plugin.ts [hunk 2]
```diff
@@ -67,6 +67,7 @@ class ImageHandler extends AbstractFileHandler {
   protected createWidget(model: IContentsModel): Widget {
     let ext = model.path.split('.').pop();
     var widget = new Widget();
+    widget.node.tabIndex = 1;
```
Comment (jasongrout) [inline src/imagehandler/plugin.ts]: `= 0`, so we aren't messing with the tabindex ordering.

**Comment by blink1073** [comment]
Comments addressed.

**Comment by dwillmer** [comment]
:+1:

**Comment by blink1073** [comment]
noice

**Pull request merged**